### PR TITLE
Update Elastic module documentation

### PR DESCRIPTION
**Pull request by dragoangel**
Update documentation for Elastic module based on latest version from PR https://github.com/rspamd/rspamd/pull/5186

### CODE REVIEW

File: doc/modules/elastic.md [hunk 1]
```diff
@@ -5,44 +5,108 @@ title: Elasticsearch module
...
+  autodetect_max_fail = 30;
+  # override works only if autodetect is disabled
+  override = {
+    name = "opensearch";
+    version = "2.17";
+  }
+};
+limits = {
+  max_rows = 500; # max logs in one bulk req to elastic and first reason to flush buffer to elastic
+  max_interval = 60; # seconds, if first log in buffer older then interval - flush buffer
```
[the remark this answers is not in the document]
Comment (dragoangel) [reply doc/modules/elastic.md:51]: this comment in the conf file, maybe better then commit it there too?

**Review by vstakhov — APPROVED** [review]

**Comment by vstakhov** [comment]
We will also need a note in the upgrading document for this change.

**Review by moisseev — APPROVED** [review]

**Comment by dragoangel** [comment]
Hm, not what auto tests are fails

**Comment by vstakhov** [comment]
> Hm, not what auto tests are fails

I think it's a bug in CI itself.

**Comment by dragoangel** [comment]
> > Hm, not get why auto tests are failed
> 
> Which autotests? check-broken-links? There are indeed broken links, but they are not related to this PR.

Yep

**Pull request merged**